terraform/provisioning/common.bash

```diff
@@ -30,6 +30,7 @@ EOF
 
 common::caddy_run() {
   sudo mkdir -p /var/lib/caddy/dotcaddy
+  sudo docker rm -f caddy || true
```

Comment:
It would be a good idea to add comments on why we remove the container when the function is named `caddy_run`. It would be confusing for a new developer reading this code without the context.

Reply:
done